tests/test_signature_coherence.py

```diff
@@ -1,31 +0,0 @@
-from crashsimilarity import crash_similarity
-
-
-if __name__ == '__main__':
```

Comment:
No need for this and two next tests while nobody will check this code "manually"